src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const changeTable = Array.from(table.rows);
```

Comment:
Consider checking if 'table.rows' is not empty to prevent errors when converting to an array.